docs/en/index.md

```diff
@@ -94,6 +105,24 @@ class StaffMember extends DataObject implements RegistryDataInterface
 }
 ```
 
+You can also reference nested fields by using dot deliniated syntax.
+
+```
+    private static $has_one = [
+        'Region' => Vendor\Package\MyRegion::class
```

Comment:
In this example you can just use `MyRegion::class` since it belongs to the same namespace, assuming that's what the example should be. This would actually be interpreted as `Vendor\Package\Vendor\Package\MyRegion`.

Reply:
Bump